config.example/inventory

```diff
@@ -53,6 +53,12 @@ kube-node
 slurm-master
 slurm-node
 
+[nfs-server:children]
```

Comment:
I've really been trying to avoid this, it's just another group people have to configure and add nodes to, and now you can't arbitrarily run against nodes not defined here.

Reply:
Hmm, fair enough -- I didn't recall the earlier change. Let me think about a better way to handle this.

Reply:
Ok, let me know what you think of the new solution